tslint.json

```diff
@@ -4,17 +4,32 @@
     "codelyzer"
   ],
   "rules": {
```

Comment:
Should we switch this to a eslint file since tslint is deprecated?
Also wonder if we should add prettier or something for formatting the styling (since probably everyone is automating this now anyways)

Reply:
I think that's a good call https://dev.to/gsarciotto/migrating-and-configuring-eslint-with-angular-11-3fg1